client/reader/site-subscriptions-manager/subscriptions-manager-wrapper.tsx

```diff
@@ -102,7 +88,7 @@ const SubscriptionsManagerWrapper = ( {
 							{ ellipsisMenuItems }
 						</SubscriptionsEllipsisMenu>
 					) }
-				</HStack>
+				</NavigationHeader>
```

Comment:
This doesn't look right, earlier the <NavigationHeader> was self closing, with an <HStack> wrapped around it. I think it should probably still be self closing?

It looks alright but just checking that it is neccessary

Reply:
This is intentional, `NavigationHeader` takes control over title, subtitle, breadcrumbs and children are expected to be any cta's you want rendered on the right of the formatted header.